packages/iTwinUI-react/package.json

```diff
@@ -0,0 +1,148 @@
+{
```

Comment:
could have been better that all old itwinui-react files were shown as moved and new ones created for monorepo...

Reply:
As I said in the [other comment](https://github.com/iTwin/iTwinUI-react/pull/613#discussion_r846291353), I can't influence the way git detects what's changed vs added.

From what I understand, if there is a new file of the same name as the one that was moved, then the diff will be messed up.